packages/client-core/src/common/services/FileBrowserService.ts

```diff
@@ -48,7 +54,17 @@ export const FileBrowserService = {
     return client.service('file-browser').update(null, { oldName, newName, oldPath, newPath, isCopy })
   },
   deleteContent: async (contentPath, type) => {
-    return client.service('file-browser').remove(contentPath, { query: { type } })
+    await client.service('file-browser').remove(contentPath, { query: { type } })
+    if (type === 'file') {
+      const staticResource = await client.service('static-resource').find({
```

Comment:
we want to implement this in the file-browser service on the backend rather than the front end, such that it is isomorphic for all file-browser requests